RabbitMQ.Stream.Client/RouteQueryResponse.cs

```diff
@@ -0,0 +1,47 @@
+﻿// This source code is dual-licensed under the Apache License, version
+// 2.0, and the Mozilla Public License, version 2.0.
+// Copyright (c) 2007-2023 VMware, Inc.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+
+namespace RabbitMQ.Stream.Client;
+
+public struct RouteQueryResponse : ICommand
+{
+    public const ushort Key = 0x0018;
```

Comment:
See https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/blob/main/RabbitMQ.Stream.Client/Client.cs#L377-L382

Reply:
So you "fix" the problem by adding a safe guard in `HandleIncoming`, but that doesn't make this constant correct IMHO. The code will work as expected, but this constant is confusing.

Reply:
Spoke to Gabriele, we are not going to change every other response struct to correct this, since the code works well and this would be a matter of neat-pick "correctness"